code/datums/wounds/pierce.dm

```diff
@@ -153,6 +157,12 @@
 	status_effect_type = /datum/status_effect/wound/pierce/moderate
 	scar_keyword = "piercemoderate"
 
+/datum/wound/pierce/moderate/update_descriptions()
+	. = ..()
```

Comment:
You don't have to call parent on these, since the parent is empty and meant to be overridden